practice-app/back-end/routes/filter_user_tweets.js

```diff
@@ -0,0 +1,46 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET 'count' amount of tweets about 'keyword' from user 'username'  */
+router.get('/', function(req, res, next) {
+
+    //initiliaze twitter object with token from key.js
+    var Twitter = require('twitter');
+    var key = require('./keys.js');
+    var client = new Twitter(key);
+
+    //convert request parameters to get parameters
+    username = req.query.user;
+    vendor = req.query.mentioned;
+    paramq = 'from:' + username + ' @' + vendor;
+
+    //paramater to be used in get method
+    var params = {
+        q: paramq,
+        count: req.query.count
+    }
+
+    //initialize empty list to collect data
+    tweetList = [];
```

Comment:
it should be better to initialize, where you used (inside the brackets in the bellow)